### PR TITLE
Fix #703 dev-tools select all attributes

### LINKED ISSUE #703: Add `Select all/none` buttons to rules constructor
Sometimes too many check-boxes are selected by default

image

### PR DESCRIPTION
**Pull request by Mizzick**
#703

### CODE REVIEW

**Comment by ameshkov** [comment]
@Mizzick when you change anything UI-related, plz attach a screenshot to the pull request

**Comment by Mizzick** [comment]
ok
screenshot 2017-05-30 11 40 09

**Comment by ameshkov** [comment]
Looks great!:)

File: Extension/browser/chrome/lib/pages/devtools-elements-sidebar.js [hunk 1]
```diff
@@ -116,6 +116,15 @@ var browser = window.browser || chrome;
             updatePanelElements();
             updateRule();
         });
+
+        document.getElementById("select-attributes-checkbox").addEventListener("click", function (e) {
+            var checked = e.currentTarget.checked;
```
Comment (atropnikov) [inline Extension/browser/chrome/lib/pages/devtools-elements-sidebar.js]: Clicking on this checkbox doesn't affect the rule text.

Comment (Mizzick) [inline Extension/browser/chrome/lib/pages/devtools-elements-sidebar.js]: that's right. Fixed.

**Pull request merged**